e2e/tests/gatekeeperSubmitsAllocationDecision_test.js

```diff
@@ -1,21 +1,25 @@
 const config = require('../config.js');
 
 let caseId;
-caseId = '1567521062611916';
 
-Feature('Application draft (populated draft)');
+Feature('Gatekeeper enters allocation decision after submission');
```

Comment:
So if we are adding hearing details in some e2e I guess we are doing something similar, could we merged the files together? gatekeeperAdministersCaseAfterSubmission_test.js the other file, I think we should have one.

Reply:
Agreed I was thinking that when doing the tests for gatekeeping I think we should try and have one suite as I was confused why there were 2 allocation tests in seperate files.